shopify/product-reviews-template/.template/docs/setup.md

```diff
@@ -0,0 +1,9 @@
+# Getting started
+
+A list of steps that you should follow:
+
+1. [Connect to Shopify](https://docs.gadget.dev/guides/tutorials/connecting-to-shopify#connecting-to-shopify) using our built-in Shopify plugin.
+
+2. Sync the code locally using `ggt` ([Gadget CLI](https://docs.gadget.dev/reference/ggt#ggt-reference)) and run `yarn dev` to test the application. Note that you're require to have a `shopify.app.toml` to run `yarn dev`. Make sure to install the [Shopify CLI](https://shopify.dev/docs/api/shopify-cli#installation) using these instructions.
+
+3. Install the application on a development store, and add the extension block to the product page. When testing, make sure that you're always running `yarn dev`.
```

Comment:
We need more instructions on how to actually set up and test the app. We should be covering how to actually generate a review and preview it on a product